examples/js/postprocessing/AdaptiveToneMappingPass.js

```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```

Comment:
Changing to RGBA shouldn't affect anything so, yeah, it's fine if it gets around the FF issue. Not sure off the top of my head whether this limitation is part of the WebGL standard or just an FF thing.